content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,19 +46,22 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-<!-- When merging this with the dev-1.27 branch conflicts will arise. The text
-as it is in the dev-1.27 branch should be used. -->
 In addition, support is needed in the 
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
-to use this feature with Kubernetes stateless pods:
+to use this feature with Kubernetes pods:
 
 * CRI-O: version 1.25 (and later) supports user namespaces for containers.
 
-Please note that containerd v1.7 supports user namespaces for containers,
-compatible with Kubernetes {{< skew currentPatchVersion >}}. It should not be used
-with Kubernetes 1.27 (and later).
+containerd v1.7 is not compatible with the userns support in Kubernetes v1.27 to v{{< skew latestVersion >}}.
```

Comment:
?
What does this mean?

Reply:
Let me try understand the situation.
containerd 1.7 + k8s 1.25/1.26 -> worked;
containerd 1.7 + k8s 1.27 -> this doesn't work;

If my understanding is correct, how about adjust the order of the two statements?

Reply:
@tengqm Your understanding is correct. But can you make a specific suggestion here on github? I mean, just changing the order doesn't make much sense IMHO, as the second sentence highlights that some version *do* support it. It would be weird to say that before sayin some version does NOT support it.

I can do the change that you want, but give me github suggestion to avoid misunderstandings :)